sdk/cosmosdb/cosmos/src/utils/hashing/v1.ts

```diff
@@ -65,6 +66,7 @@ function encodeByType(key: v1Key) {
       }
       return Buffer.from(BytePrefix.Undefined, "hex");
     case "undefined":
+    default:
```

Comment:
These feel like a bit of indirection since we have the undefined case there but empty. Maybe redundant to include it in both cases but I might prefer that for future devs

Reply:
looking back at this, `default` should throw an error that an incorrect type has been passed